dss/operations/params.py

```diff
@@ -0,0 +1,373 @@
+"""
+Get, set, and unset environment variables in the SSM store and in deployed lambda functions
+"""
+import boto3
+import os
+import sys
+import typing
+import argparse
+import json
+import logging
+import select
+
+from dss.operations import dispatch
+from dss.util.aws.clients import ssm as ssm_client  # type: ignore
+from dss.util.aws.clients import es as es_client  # type: ignore
+import dss.util.aws.clients
+
+lambda_client = getattr(dss.util.aws.clients, "lambda")
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_ssm_prefix():
+    """
+    Use information from the environment to assemble
+    the necessary prefix for accessing variables in
+    the parameter store.
+    """
+    store_name = os.environ["DSS_PARAMETER_STORE"]
+    stage_name = os.environ["DSS_DEPLOYMENT_STAGE"]
+    store_prefix = f"{store_name}/{stage_name}"
+    return store_prefix
+
+
+def get_ssm_lambda_environment(prefix):
```

Comment:
I had initially designed this command with a `--stage` flag, so that users could specify the stage on the command line. After some feedback from the team, I dropped the `--stage` flag. But am realizing that I did not get rid of requiring callees to specify a prefix (which would change if the user specified a `--stage` that was different from the one in the environment, but it will not change because it always gets the prefix from the environment). So I need to remove this `prefix` arg from this and all other functions and have them call [the function Lon mentioned above](https://github.com/HumanCellAtlas/data-store/pull/2324/files#r317287921).